settings/checkstyle/checkstyle-suppressions.xml

```diff
@@ -11,7 +11,9 @@
   <!-- Suppress all checks in generated sources. -->
   <suppress checks=".*" files="[\\/]gen-src[\\/]" />
   <!-- Suppress checks in large forks to make diffing against upstream easier. -->
-  <suppress checks=".*" files="[\\/]grpc[\\/]ServerCallImpl.java$" />
+  <suppress checks=".*" files="[\\/]common[\\/]MediaType.java$" />
+  <suppress checks=".*" files="|[\\/]internal[\\/]DefaultAttributeMap.java" />
+  <suppress checks=".*" files="[\\/]reactive[\\/]DefaultSslInfo.java" />
```

Comment:
Could we only suppress `FinalLocalVariable` for these files?

Reply:
For forks, isn't it better to not apply any of our style at all for easier diffs?